### PR TITLE
Asset tracker: UI module fixes

### PR DESCRIPTION
**Pull request by bjda**
applications: asset_tracker: Fix typo in UI

Fixes a typo in the UI library restricting the use of the external pins.

---
applications: asset_tracker: Fix PWM/GPIO consistency in UI

The UI library passes an initial period to the PWM driver, and
calls both the PWM and GPIO driver when used in non-PWM mode.
This ensures consistent output when the PWM hardware is enabled
and disabled at run-time.

### CODE REVIEW

File: applications/asset_tracker/src/ui/nmos.c [hunk 1]
```diff
@@ -184,6 +194,17 @@ int ui_nmos_init(void)
 		return -ENODEV;
 	}
 
+	for (size_t i = 0; i < ARRAY_SIZE(nmos_pins); i++) {
+		pwm_pin_set_usec(
+			pwm_dev,
```
Comment (rlubos) [inline applications/asset_tracker/src/ui/nmos.c]: We typically don't start the argument list in a new line (unless we're constrained by the char limit).  See line 95.

File: applications/asset_tracker/src/ui/nmos.c [hunk 2]
```diff
@@ -232,5 +253,9 @@ int ui_nmos_write(size_t nmos_idx, u8_t value)
 		return err;
 	}
 
-	return err;
+	return pwm_out(
+		nmos_pins[nmos_idx].pin,
```
Comment (rlubos) [inline applications/asset_tracker/src/ui/nmos.c]: Same here.

File: applications/asset_tracker/src/ui/nmos.c (hunk 1 above)
Comment (bjda) [reply applications/asset_tracker/src/ui/nmos.c]: Should be fixed now

File: applications/asset_tracker/src/ui/nmos.c (hunk 2 above)
Comment (bjda) [reply applications/asset_tracker/src/ui/nmos.c]: Should be fixed now

**Comment by hmhalvorsen** [comment]
LGTM

**Comment by rlubos** [comment]
@bjda I have to ask you to sign the CLI again before I can merge. The CI seems not to respect the old ones.

**Pull request merged**